<html>

<head>
<title>About Rahnuma</title>
</head>

<body>

<h1 align="center">Rahnuma: Pathway Prediction and Network Analysis Tool</h1>
<h2 align="center">About</h2>
<table border="1" cellpadding="4" cellspacing="0" style="border-collapse: collapse; border-left-width: 0; border-top-width: 0; border-bottom-width: 0" bordercolor="#FFFFFF" width="100%">
  <tr>
    <td width="100%" style="border-style: none">
<p>Rahnuma is a joint project between <a href="http://www.stats.ox.ac.uk/research/genome" target="_blank">Hein Group</a> from <a href="http://www.stats.ox.ac.uk" target = "_blank">Department of Statistics</a> and <a href="http://dps.plants.ox.ac.uk/plants/staff/GailPreston.aspx" target="_blank">Preston Group</a> from <a href="http://www.plants.ox.ac.uk" target="_blank">Department of Plant Sciences</a> at the <a href="http://www.ox.ac.uk" target="_blank">University of Oxford</a> and is being developed as a part of <a href="http://www.stats.ox.ac.uk/~mithani" target="_blank">Aziz Mithani</a>'s DPhil. The name <i>Rahnuma</i> comes from the Urdu language and literally means &ldquo;someone who guides through the path&rdquo;. 

<p>Rahnuma is a tool for prediction and analysis of metabolic pathways and comparison of metabolic networks that represents metabolic networks as hypergraphs and computes all possible pathways between two or more metabolites. Rahnuma provides an intuitive way to answer different biological questions focusing on differences between multiple organisms or evolution of different species by allowing pathway based metabolic network comparisons at organism as well as phylogenetic level.

<p> Rahnuma uses hypergraphs to represent metabolic networks internally. Hypergraphs have advantages over ordinary graphs for representing metabolic networks. Hypergraphs can capture the relationships between multiple metabolites in a reaction and also allow enumeration of multiple reactions connecting metabolites. They also provide an intuitive approach to finding pathways between two
metabolites by regarding a pathway as a series of connected hyperedges. 

<p> Rahnuma consists of three main modules, namely Pathway Analysis, Comparative Analysis and Network Analysis. The tool can be used to perform a wide range  of analyses ranging from simple pathway prediction to identifying reactions that are exclusive to a pathway or act as a bridge in a pathway, and from simple network comparison to phylogeny based pathway and comparative analyses. The tool is written in Java and uses data from <a href="http://www.genome.jp/kegg" target="_blank">KEGG</a> relating to the <a href="http://www.genome.jp/kegg/pathway.html" target="_blank">pathway maps</a> involved in metabolism in addition to the manually curated carbon and nitrogen connections for pathway maps focusing on nutrient assimilation and energy metabolism. The list of curated pathways present in Rahnuma can be found <a href="http://portal.stats.ox.ac.uk/userdata/rahnuma/files/curatedpathways.txt" target="_blank">here</a>. Also, the list of organisms available in Rahnuma can be found <a href="http://portal.stats.ox.ac.uk/userdata/rahnuma/files/rahnumaorganisms.txt" target="_blank">here</a>.

<p>Rahnuma is a versatile tool and can be used for many different applications including predicting amino acid assimilation pathways, analysing pathways and identifying reactions at different levels of a phylogeny, comparing host and pathogen networks, and identifying effective knock-out targets by finding bridge reactions in a network etc. It can be used in to explain the experimental results and also to answer specifically biological questions. By performing <i>in silico</i> analysis and comparison, Rahnuma can be used as a hypothesis generating tool which may prove very useful in further analysis and designing experiments.
    </td>
  </tr>
  <tr>
    <td width="100%" style="border-style: none">
Some worked examples are available <a href="tutorials.html">here</a>. For enquiries or comments/suggestions relating to Rahnuma project, please contact Aziz Mithani at <b>mithani [at] stats [dot] ox [dot] ac [dot] uk</b>. 
    </td>
  </tr>
  <tr>
    <td width="100%" style="border-style: none">
	<p/>
	<p>A useful list of additional pathway prediction and comparison tools has been compiled by Karoline Faust and is available <a href="http://www.bigre.ulb.ac.be/Users/karoline/links.php" target="_blank">here</a>.</p>
    </td>
  </tr>
  <tr>
    <td width="100%" style="border-style: none">
<p></p>
<p></p>
<b>Legal stuff:</b><br/>
Rahnuma is free for non-commercial use by academicusers. Commercial users should contact us for further information. Commercial users should also read the <a href="http://www.genome.jp/kegg/legal.html" target="_blank">copyright</a> information on the KEGG website before using Rahnuma.
    </td>
  </tr>
</table>
<p></p>

</body>

</html>